test/18-startup.ts

```diff
@@ -0,0 +1,16 @@
+import { Supervisor } from '../src/supervisor';
+import { expect } from './lib/chai-config';
+
+describe('Startup', () => {
+	it('should startup correctly', function() {
+		const supervisor = new Supervisor();
+		expect(supervisor.init()).to.not.throw;
+		// Cast as any to access private properties
+		const anySupervisor = supervisor as any;
```

Comment:
I'm not a massive fan of this cast but it's probably preferable to making all the properties public